Makefile

```diff
@@ -0,0 +1,18 @@
+BINARY_FOLDER_PREFIX?=/usr/local
+BINARY_FOLDER=$(BINARY_FOLDER_PREFIX)/bin/
+SWIFT_BUILD_FLAGS=-c release -Xswiftc -static-stdlib
+
+.PHONY: clean build install uninstall
+
+clean:
+	cd Generator && swift package clean
```

Comment:
nit: `Generator` is in 3 places.  Maybe a `GENERATOR_SUBDIR` ?

Reply:
Good point. I'll update that.